scripts/hudson/narayana.sh

```diff
@@ -480,27 +480,36 @@ function osgi_tests {
 }
 
 function xts_as_tests {
-  echo "#-1. XTS AS Integration Test"
+  echo "#-1. XTS AS Tests"
   cd ${WORKSPACE}/jboss-as
+  ./build.sh -f xts/pom.xml -B $IPV6_OPTS -Dtimeout.factor=300 -Dsurefire.forked.process.timeout=12000 -Dsurefire.extra.args='-Xmx512m' -Djboss.dist="$JBOSS_HOME" -Dversion.org.jboss.narayana=${NARAYANA_CURRENT_VERSION} "$@" test
```

Comment:
Is it testing the transformer? I don't think I intended that. I think it would be testing the XTS WildFly subsystem with the latest version of XTS (and not specifically about transformation)

Reply:
I understand now that the transformer comment refers WildFly transformer: https://github.com/wildfly/wildfly/blob/main/xts/src/test/java/org/jboss/as/xts/XTSSubsystemTestCase.java (not Jakarta EE). Thanks for pointing this out.

A note that `transactions` tests does include a functional test: https://github.com/wildfly/wildfly/blob/main/transactions/src/test/java/org/jboss/as/txn/TestWildFlyTSR.java